install-dev/classes/controllerConsole.php

```diff
@@ -91,7 +91,12 @@ final public static function execute($argc, $argv)
             echo 'Arguments available:' . PHP_EOL;
             foreach ($available_arguments as $key => $arg) {
                 $name = isset($arg['name']) ? $arg['name'] : $key;
-                echo '--' . $name . "\t" . (isset($arg['help']) ? $arg['help'] : '') . (isset($arg['default']) ? "\t" . '(Default: ' . $arg['default'] . ')' : '') . PHP_EOL;
+                echo '--'
+                    . $name
+                    . "\t"
+                    . (isset($arg['help']) ? $arg['help'] : '')
+                    . (isset($arg['default']) ? "\t" . '(Default: ' . (is_array($arg['default']) ? '[' . implode(',', $arg['default']) . ']' : $arg['default']) . ')' : '')
+                    . PHP_EOL;
```

Comment:
Good but I think It's a little difficult to read (two ternary operator in one line), I purpose to transform the code like this :

```suggestion
$help = isset($arg['help']) ? $arg['help'] : '';
$default = '';

if(isset($arg['default'])) {
    if(is_array($arg['default'])) {
        $default = '(Default: [' . implode(',', $arg['default']) . '])';
    } else {
        $default = '(Default: ' . $arg['default'] . ')';
    }
}

echo '--' . $name . "\t" . $help . "\t" . $default . PHP_EOL;
```

What do you think about it ? :)